dataset_test.go

```diff
@@ -748,6 +770,10 @@ func (me *datasetTest) TestLiteralExpressionMap() {
 	assert.Equal(t, buf.String(), `("a" NOT IN ('a', 'b', 'c'))`)
 	assert.NoError(t, ds.Literal(me.Truncate(buf), Ex{"a": Op{"is": nil, "eq": 10}}))
 	assert.Equal(t, buf.String(), `(("a" = 10) OR ("a" IS NULL))`)
+	assert.NoError(t, ds.Literal(me.Truncate(buf), Ex{"a": Op{"between": RangeVal{1,10}}}))
```

Comment:
`golint` gripes about using `RangeVal{1,10}` this way. I would specify the `Start` and `End` like you do in https://github.com/doug-martin/goqu/pull/25/files#diff-7101a3ea121adf8d5b0acb5a2c520ca3R1524